lang/en/local_lsf_unification.php

```diff
@@ -89,6 +89,8 @@
 $string['config_category'] = "Category";
 $string['config_category_wish'] = "Category Relocation Wish";
 $string['config_category_wish_help'] = "If you have a wish to get your course moved into a more specific category, please leave a comment here containing your wish-category and path.";
+$string['config_course_semester'] = "Semester";
+$string['config_course_semester_missing'] = "The field defining the current semester is missing.";
```

Comment:
*Semester -> Term
*the current semester -> the term that the course belongs to

Reply:
https://github.com/NinaHerrmann/his_unification/blob/5d18a0fe3f1b46caeae24a3908db44e9c2e85fef/lang/en/local_lsf_unification.php#L92